packages/api/src/providers/fitbit.ts

```diff
@@ -332,4 +336,54 @@ export class Fitbit extends Provider implements OAuth2 {
       }
     );
   }
+
+  async postAuth(token: string, userId?: string) {
+    const accessToken = JSON.parse(token).access_token;
+
+    const scopes = JSON.parse(token).scope;
+
+    const subscriptionTypes: Record<
+      FitbitScopes,
+      Omit<FitbitCollectionTypes, "userRevokedAccess">
```

Comment:
I was checking this code local and realized `Omit` doesn't work for enums. :/

Sorry, but TIL. We can use `Exclude` in such cases.

While here, wouldn't be useful to subscribe to `userRevokedAccess` and just disregard it until we're ready to implement #652? Just b/c this would make types easier - no need to be fiddling with subsets of possible types.

If we keep it like this, maybe we create a type to represent `Exclude<FitbitCollectionTypes, "userRevokedAccess">`, so we have it defined in only one place?